lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -889,6 +889,16 @@ void PatchEntryPointMutate::setFuncAttrs(Function *entryPoint) {
     spiPsInputAddr.bits.sampleCoverageEna = builtInUsage.sampleMaskIn;
 
     builder.addAttribute("InitialPSInputAddr", std::to_string(spiPsInputAddr.u32All));
+
+    bool hasDepthExport = builtInUsage.sampleMask || builtInUsage.fragStencilRef || builtInUsage.fragDepth;
+    builder.addAttribute("amdgpu-depth-export", std::to_string(hasDepthExport ? 1 : 0));
```

Comment:
nit: why not `hasDepthExport ? "1" : "0"`?

Reply:
Thanks